demo/index.html

```diff
@@ -16,9 +16,25 @@ <h2 class="accordion-label">Accordion Heading</h2>
 				<p>here the content of 1st tab <a href="#">link</a></p>
 			</div> <!-- //.accordion-content -->
 
-			<button class="accordion-header" type="button">Accordion Header</button>
+			<button class="accordion-header" type="button">Accordion Header with Nested Accordion</button>
 			<div class="accordion-content">
 				<p>here the content of 2nd tab <a href="#">link</a></p>
```

Comment:
@kdo 
Only thing I'd change is adding  something like `<h2 class="accordion-label" tabindex="-1">Parent Accordion Heading</h2>` here for consistency.

Reply:
Thanks @joesnellpdx added!